### PR TITLE
dataviz pages

### PR DESCRIPTION
**Pull request by greeshmareddyg**
story: https://github.ibm.com/brand/design-language/issues/87

### CODE REVIEW

**Comment by vpicone** [comment]
@mjabbink should we be using “Data Vis” here? We call it ”Data Visualization” in Carbon.

**Comment by mjabbink** [comment]
Should be the same whichever it is. “Data visualization” is probably the way to go

**Comment by vpicone** [comment]
What’s the status on this? Could you make the update to read Data visualization and add alt tags to the images?

**Comment by mjabbink** [comment]
yes, page can rad Data visualization @sadekbazaraa what do think. Staying consistent with Carbon is good or we both change to Data vis which seems odd to me.

**Comment by sadekbazaraa** [comment]
@mjabbink agreed. It should be "Data visualization" with a lower case v. Right now the "V" is capped.

**Comment by mjabbink** [comment]
yes, lowercase “v” for visualization

**Comment by vpicone** [comment]
Looking good! We do still need alt tags describing these images though. Name of the chart/variation. Something along those lines.

File: src/styles/data-vis.scss [hunk 1]
```diff
@@ -15,7 +15,7 @@
 }
 
 .axesntricks-images{
-  margin-top: -60px;
+  margin-top: -60px !important;
```
Comment (vpicone) [inline src/styles/data-vis.scss]: We try to avoid using important wherever possible.

Comment (vpicone) [reply src/styles/data-vis.scss]: If it’s not possible, a line comment with an explanation will do.

**Comment by greeshmareddyg** [comment]
Actually, I was testing and not using it, removing.

File: src/styles/data-vis.scss (hunk 1 above)
Comment (greeshmareddyg) [reply src/styles/data-vis.scss]: Actually, I was testing and not using it, removing that style.

File: src/components/OverviewCard/overview-card.scss [hunk 2]
```diff
@@ -0,0 +1,79 @@
+$active-ui: #bebebe;
+$hover-ui: #e5e5e5;
```
Comment (sstrubberg) [inline src/components/OverviewCard/overview-card.scss]: This file is using a lot of hex colors. Can we use the carbon-colors?

File: src/pages/data-visualization/charts.mdx [hunk 3]
```diff
@@ -0,0 +1,335 @@
...
+    title="Bubble chart"
+    href="https://www.carbondesignsystem.com/data-visualization/basic-charts#bubble-chart"
+    >
+
+![bubble chart](/images/charts/Comparisons/Bubble_chart.svg)
+
+</OverviewCard>
+</Column>
+
+<Column  colMax={3} colXl={3} colLg={4} colMd={4} colSm={2} noGutterSm >
```
[the remark this answers is not in the document]
Comment (greeshmareddyg) [reply src/pages/data-visualization/charts.mdx]: yeah, I have used this code and styles exactly from the carbon design, to replicate it. I can make this change here, can you make this change there as well.

**Comment by sstrubberg** [comment]
@greeshmareddyg what am i reviewing here? The issue that you link to is busted

**Comment by greeshmareddyg** [comment]
> @greeshmareddyg what am i reviewing here? The issue that you link to is 404'ing on me.

@sstrubberg could you please check if you are able to see now.

**Comment by sstrubberg** [comment]
> > @greeshmareddyg what am i reviewing here? The issue that you link to is 404'ing on me.
> 
> @sstrubberg could you please check if you are able to see now.

still busted.

**Comment by greeshmareddyg** [comment]
> @greeshmareddyg can you remove any unused images from this PR? I could be wrong but I see 85 pngs, 100 svgs, and 17 videos. It doesn't appear that you're actually using all of them. Please let me know if this is incorrect though!

@vpicone removed the unused files.

**Comment by vpicone** [comment]
@sadekbazaraa I didn't see any comments here, was this good to merge?

**Comment by sadekbazaraa** [comment]
@vpicone the only thing we seem to be missing still is the 'launch external site' icon in the lower right corner of the chart images so the user understands they're leaving the page. If that's something simple enough to do before we release then let's add. If it will take some time, then we can release now and prioritize that as the next fix.

Screen Shot 2020-05-15 at 3 42 47 PM

FYI @mjabbink

**Comment by vpicone** [comment]
@sadekbazaraa yeah the issue for that exists here: https://github.com/carbon-design-system/design-language-website/issues/518

**Comment by mjabbink** [comment]
Then we can launch but let’s prioritize #518

**Comment by vpicone** [comment]
> Then we can launch but let’s prioritize #518

Sounds good, chuck it up on the brand prio.

**Pull request merged**